app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -211,9 +217,16 @@ protected static void createTestShopList() throws ExecutionException, Interrupte
         Task<DocumentReference> t = FirestoreShopList.storeNewShopList(db.collection("shop_lists"), shopList, household);
         Tasks.await(t);
         shopList.setOnlineReference(t.getResult());
+
+        household = db.collection("households").document(TEST_HOUSEHOLD_NAMES[1]);
+        shopList = new FirestoreShopList(household);
+        shopList.addItem(TEST_ITEM);
+        t = FirestoreShopList.storeNewShopList(db.collection("shop_lists"), shopList, household);
```

Comment:
Actually it is not necessary to add a shopList to a new house as it is added anyway (without item) if it does not exist by the MainScreen. However you might leave it like that so this  house already has an item